apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -206,7 +206,7 @@ class ProgressLesson extends React.Component {
           )}
         </div>
         {showTeacherInfo && viewAs === ViewType.Teacher && (
-          <ProgressLessonTeacherInfo lesson={lesson} />
+          <ProgressLessonTeacherInfo lesson={lesson} shareUrl={levels[0].url} />
```

Comment:
FYI: shareUrl generally refers to the unique url of a user's project. i.e. 
https://studio.code.org/projects/applab/pM1f-UJm2YPi_MqHIpTMbaKgFrJGgifalAd1eaJrIhU
would be a shareUrl (accessed by `dashboard.project.getShareUrl()`). I'd recommend renaming this to something like `levelUrl` to clear up confusion.

Reply:
👍